sale_order_line_variant_description/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2015 Alex Comba - Agile Business Group
```

Comment:
Please replace by
```
# Copyright 2015-2017 Alex Comba - Agile Business Group
```
according to https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py#L2